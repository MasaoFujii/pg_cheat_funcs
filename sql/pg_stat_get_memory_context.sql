CREATE EXTENSION pg_cheat_funcs;

\pset null '(null)'

SELECT name FROM pg_stat_get_memory_context() WHERE parent IS NULL;

DROP EXTENSION pg_cheat_funcs;
