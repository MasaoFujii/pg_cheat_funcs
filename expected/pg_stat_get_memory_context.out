CREATE EXTENSION pg_cheat_funcs;
\pset null '(null)'
SELECT name FROM pg_stat_get_memory_context() WHERE parent IS NULL;
       name       
------------------
 TopMemoryContext
(1 row)

DROP EXTENSION pg_cheat_funcs;
